hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -157,15 +159,48 @@ def setUp(self):
 
         s.set_elements(elements)
         s.add_lines(xray_lines)
+        s.axes_manager[0].scale = 0.5
+        s.axes_manager[1].scale = 0.5
         self.signal = s
 
     def test_quant_lorimer(self):
         s = self.signal
+        method = 'CL'
         kfactors = [1, 2.0009344042484134]
+        composition_units = 'weight'
         intensities = s.get_lines_intensity()
-        res = s.quantification(intensities, kfactors)
-        nt.assert_true(np.allclose(res[0].data, np.array(
-                       [22.70779, 22.70779]), atol=1e-3))
+        res = s.quantification(intensities, method, kfactors, composition_units)
+        nt.assert_true(np.allclose(res[0].data, np.array([22.70779, 22.70779],
+                    [22.70779, 22.70779]), atol=1e-3))
+
+    def test_quant_zeta(self):
+        s = self.signal
+        method = 'zeta'
+        compositions_units = 'weight'
+        factors = [20, 50]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors, compositions_units)
+        nt.assert_true(np.allclose(res[1].data, array(
+                [[  16.216216216,   16.216216216],
+                [  16.216216216,   16.216216216]]), atol=1e-3))
+        nt.assert_true(np.allclose(res[0][1].data, array(
+                [[  2.37122140e-06,   2.37122140e-06],
+                [  2.37122140e-06,   2.37122140e-06]]), atol=1e-3))
+
+    def test_quant_cross_section(self):
+        s =self.signal
+        method = 'cross_section'
+        factors = [3, 5]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors)
+        (np.allclose(res[1][1].data, np.array(
+            [[ 16.02176621,  16.02176621],
+            [ 16.02176621,  16.02176621]])), atol=1e-3)
+        nt.assert_true(np.allclose(res[1][1].data, np.array(
+           [[ 19.8669901,  19.8669901],
+           [ 19.8669901,  19.8669901]])), atol=1e-3)
+        nt.assertTrue(np.allclose(results1[0].data, np.array(
```

Comment:
Agreed. Just trying to work out why my unit tests were failing. 

> On 07 Apr 2016, at 10:16, Duncan Johnstone <notifications@github.com> wrote:
> 
> In hyperspy/tests/signal/test_eds_tem.py <https://github.com/hyperspy/hyperspy/pull/913#discussion_r58834668>:
> 
> > +                [[  2.37122140e-06,   2.37122140e-06],
> > +                [  2.37122140e-06,   2.37122140e-06]]), atol=1e-3))
> > +
> > +    def test_quant_cross_section(self):
> > +        s =self.signal
> > +        method = 'cross_section'
> > +        factors = [3, 5]
> > +        intensities = s.get_lines_intensity()
> > +        res = s.quantification(intensities, method, factors)
> > +        (np.allclose(res[1][1].data, np.array(
> > +            [[ 16.02176621,  16.02176621],
> > +            [ 16.02176621,  16.02176621]])), atol=1e-3)
> > +        nt.assert_true(np.allclose(res[1][1].data, np.array(
> > +           [[ 19.8669901,  19.8669901],
> > +           [ 19.8669901,  19.8669901]])), atol=1e-3)
> > +        nt.assertTrue(np.allclose(results1[0].data, np.array(
> I think this should be nt.assert_true?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/hyperspy/hyperspy/pull/913/files/ff223181b7d44327f1bbf263d33e8bd9580a5748#r58834668>